packages/amplify-e2e-tests/src/__tests__/apigw.test.ts

```diff
@@ -30,7 +30,7 @@ describe('API Gateway e2e tests', () => {
     await amplifyPushAuth(projRoot);
     await addAuthWithGroupsAndAdminAPI(projRoot); // Groups: Admins, Users
     await amplifyPushAuth(projRoot);
-    await addRestApi(projRoot, { isFirstRestApi: false, path: '/foo' });
+    await addRestApi(projRoot, { isFirstRestApi: false, path: '/foo', existingLambda: true });
```

Comment:
this setting shouldn't matter because the path is set in the options.

Reply:
helper code:

```ts
  if (!isFirstRestApi) {
    chain.wait('Would you like to add a new path to an existing REST API');

    if (settings.path) {
      chain
        .sendYes()
        .wait('Select the REST API you want to update')
        .sendCarriageReturn() // Select the first REST API
        .wait('What would you like to do?')
        .sendCarriageReturn() // Add another path
        .wait('Provide a path')
        .sendLine(settings.path)
        .wait('Choose a lambda source')
        .sendKeyDown()
        .sendCarriageReturn() // Existing lambda
        .wait('Choose the Lambda function to invoke by this path')
        .sendCarriageReturn() // Pick first one
        .wait('Restrict API access')
        .sendNo() // Do not restrict access
        .wait('Do you want to add another path')
        .sendNo() // Do not add another path
        .sendEof();

      return chain.runAsync();
    } else {
      chain.sendNo();
    }
  }
  ...
  ```

Reply:
https://app.circleci.com/pipelines/github/aws-amplify/amplify-cli/4955/workflows/cd148ede-ced8-499e-82ae-d0ac6caf1a4a/jobs/115006/tests#failed-test-0